apps/backend/src/modules/kobo/constants.ts

```diff
@@ -0,0 +1,21 @@
+import { DROUGHT, FLOOD, INCIDENT } from '@wfp-dmp/interfaces';
```

Comment:
per discussion, make sure this has more functionality than repeating what happens in env.validation, maybe make sure the correct value is filled in in the env.rc